### ISSUE TITLE
Support training mode in BatchNormalization 

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#20]

**Comment by mysticatea** [comment]
I'm happy if this implemented.

**Comment by mysticatea** [comment]
I'm happy if this is implemented, in order to write an AlphaZero clone.
